doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md

```diff
@@ -28,8 +28,18 @@ YouTube Video
 YouTube Video
 {% endembed %}
 
+### Syncing your fork
+{% embed url="https://youtu.be/A2gqOvGpH_0" %}
+YouTube Video
+{% endembed %}
+
 Some other set 1 learning videos are not available yet. Check back on the 9th of October 2022.
 
+### General information about the training tasks
+{% embed url="https://youtu.be/gGpH3kdbQns" %}
+YouTube Video
+{% endembed %}
+
```

Comment:
Hi @Kesav890 
Nice work finding the right file and adding the changes in the right place.

But I cannot accept this change. The reason is that it is not advisable to make changes directly in the develop branch. There are also other things written in our guide that you didn't take note of. Just close this pull request and delete your fork and start all over.

This time, please follow the guidelines here for your new pull request to be accepted: https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/making-and-submitting-changes.

If you have any questions or are confused about anything feel free to ask me. I will gladly respond.